crates/build-utils/src/content_reader.rs

```diff
@@ -71,4 +100,31 @@ impl ContentReader {
         // Return a clone of the block
         Ok(first_block.clone())
     }
+
+    pub fn transaction(test_case: &Value, block: &Value) -> Result<Value, eyre::Error> {
```

Comment:
we can delete this then?

Reply:
we still use it inside secret_key to get the tx from a block